firebase_admin/db.py

```diff
@@ -23,16 +23,25 @@
 import collections
 import json
 import sys
+import threading
+import time
 
 import requests
 import six
 from six.moves import urllib
+from google.auth import transport
 
 import firebase_admin
 from firebase_admin import _http_client
 from firebase_admin import _utils
+from firebase_admin._sseclient import SSEClient, KeepAuthSession
```

Comment:
As a matter of style, we don't import individual classes and members. Please import the module here as `from firebase_admin import _sseclient`, and change the code below accordingly.